demo-forms/custom_contacts/README.MD

```diff
@@ -0,0 +1,11 @@
+# Applying Sample Custom Contact Types
+
+
+1. Download the current `app_settings.json`
```

Comment:
While I think it's clear you mean [this one](https://github.com/medic/cht-core/blob/master/config/default/app_settings.json) (right?), we should give them a deep link right here to make it easier.

Reply:
I'll clear this up. It is the `app_settings.json` from your current instance.